util/test/account_testutil.go

```diff
@@ -320,33 +321,142 @@ func MakeAppCallWithInnerTxn(appSender, paymentSender, paymentReceiver, assetSen
 		{
 			SignedTxn: transactions.SignedTxn{
 				Txn: transactions.Transaction{
-					Type: protocol.PaymentTx,
+					Type: protocol.ApplicationCallTx,
```

Comment:
Pedantic change - just to reflect that only `appl` calls currently support itxns